src/routes/index.svelte

```diff
@@ -73,23 +50,82 @@
     let loading = false
     let highlightedSymbolIndex = null
 
+    let isDocumentReady = false
+
     onMount(() => {
+        let DEFAULT_OPTIONS = []
+
+        if (typeof document !== 'undefined') {
+            isDocumentReady = true
+        }
+
+        selectedPriceOption = getDefaultPriceOption(price)
+
+        DEFAULT_OPTIONS = getDefaultOptions(symbols)
         fetchStockData(DEFAULT_OPTIONS)
     })
 
+    const getDefaultOptions = symbols => {
+        let symbolList
+        const storedSymbols = localStorage.getItem(LS_SYMBOLS)
+
+        if (symbols) {
```

Comment:
If we have query parameters, we should replace what is in local storage

Reply:
This gets handled by the reactive declaration when history is updated.